tests/unit/Storage/RequesterPaysTest.php

```diff
@@ -0,0 +1,430 @@
+<?php
```

Comment:
Could you tell me more about the tests? Does it use two projects for example to test UserProject? 
I'm not sure how much of the code is mocked versus not mocked given `prophesize`.

Reply:
This unit test mocks each HTTP request and inspects it for the `userProject` query string parameter. We have a separate [system test](https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/tests/system/Storage/RequesterPaysTest.php) which tests using two projects. I have not yet updated that with the methods covered in the PR. I was planning on doing that separately.

Reply:
Could you cc me on the PR?